_posts/2015-03-27-ApiBlueprint-button.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Sharing is caring - socialize you're public API blueprint
+excerpt: Learn the art of API blueprint through socialization and collaboration.
+layout: post
+date: 2015-03-27 00:00:00 +0100
+author: vincenzo
+published: true
+commets: true
+---
+
+We have recently received a lot of feedback from users that would like to learn more about API Blueprint and how to present stunning and beautiful documentation.
```

Comment:
What do you mean by 'present'? Are you trying to say, '.. design stunning and beautiful documentation.'?

Revise this sentence to clearly state the problem and why users want to learn more about API blueprint. It's not clear 'why' presenting stunning and beautiful documentation is important. Aesthetics are nice, but what really motivates developers to use / reference a great API?